Source/WebCore/PAL/pal/spi/ios/MobileGestaltSPI.h

```diff
@@ -27,8 +27,6 @@
 
 #import <wtf/Platform.h>
 
-#if PLATFORM(IOS_FAMILY)
```

Comment:
Is `MobileGestalt.h` available on macOS? I wasn't able to find it.

Reply:
Yes it is

Reply:
(I found it now)